pkg/webhook/admission/http.go

```diff
@@ -99,6 +99,6 @@ func (wh *Webhook) writeResponse(w io.Writer, response Response) {
 		wh.writeResponse(w, Errored(http.StatusInternalServerError, err))
 	} else {
 		res := responseAdmissionReview.Response
-		wh.log.V(1).Info("wrote response", "UID", res.UID, "allowed", res.Allowed, "result", res.Result)
+		wh.log.V(1).Info("wrote response", "UID", res.UID, "allowed", res.Allowed, "code", res.Result.Code, "reason", res.Result.Reason)
```

Comment:
`res.Result` might be nil if the request gets approved, if we just unconditionally access it like this we might panic

Reply:
/hold

Reply:
Ups. Good catch!